lib/src/Pos/Reporting/Production.hs

```diff
@@ -31,10 +33,17 @@ productionReporter
     -> Reporter IO
 productionReporter params diffusion = Reporter $ \rt -> withWlogTempFile logConfig $ \mfp -> do
     rt' <- extendWithNodeInfo diffusion rt
-    reportNode logTrace protocolMagic compileTimeInfo servers mfp rt'
+    (reportNode logTrace protocolMagic compileTimeInfo servers mfp rt'
```

Comment:
Are parens really needed here?

Reply:
I don't think so - I had them there to make the initial lack-of-indentation less confusing. But with your suggested indent of the bottom 2 lines, they seem unnecessary. Just force-pushed a fix.